tracing-mock/src/span.rs

```diff
@@ -47,6 +246,50 @@ impl ExpectedSpan {
         }
     }
 
+    /// Sets the target to expect when matching a span.
+    ///
+    /// If a span is recorded with a different target, this expectation
+    /// will fail.
```

Comment:
```suggestion
    /// If an event is recorded with a target that doesn't match the
    /// observed target, this expectation will fail.
```

Reply:
I like this wording, but I think that "observed" might not be the right word.

We have the "recorded target" (which is what we are testing). Then I think that what we are comparing to should be the "provided target" rather than the "observed target".

Does that make sense?

Reply:
yeah, that makes sense. "provided target" is probably best, then!